frontend/src/pages/login/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React , {FC } from 'react'
+import { Card, Button } from 'antd';
+import * as s from './login.module.css';
+
+const Login : FC<{}> = () => {
+
+    return <Card className={s['loginCard']}>
+        <h1>Group Buy</h1>
+        <Button type='primary'>Google Login</Button>
+        
+
+       
+    </Card>
```

Comment:
Is the indentation intended? It seems a little off to me. 😅

Reply:
ah i fixed this!